client/src/components/Tickets/TicketModal.js

```diff
@@ -180,7 +201,8 @@ TicketHistoryCard.propTypes = {
   created: PropTypes.instanceOf(Date).isRequired,
   creator: PropTypes.string.isRequired,
   ticketCreator: PropTypes.bool.isRequired,
-  content: PropTypes.string.isRequired
+  content: PropTypes.string.isRequired,
+  attachments: PropTypes.objectOf(PropTypes.any).isRequired
```

Comment:
Not an object, right? But an array of objects.
```suggestion
  attachments: PropTypes.array.isRequired
```